src/Serializer.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Dunglas\DoctrineJsonOdm;
+
+use Symfony\Component\Serializer\Serializer as BaseSerializer;
+
+class Serializer extends BaseSerializer
+{
+    const KEY_TYPE = '#type';
```

Comment:
Guess we can make those `private`, wdyt? :)

Reply:
I forgot it when updated minimun required PHP version. Fixed now.